app/services/sign_in/session_creator.rb

```diff
@@ -19,6 +20,12 @@ def perform
 
     private
 
+    def validate_credential_lock
+      if user_verification.locked == true
```

Comment:
just do `if user_verification.locked` here, comparing to `true` is implicit as a 'truthy' comparison here